test/fixed_width_rw_error.js

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) 2015 Uber Technologies, Inc.
```

Comment:
this is a struct test case, not fixed_width_rw specific; please move it into a case in test/struct_rw